llvm/test/Other/lit-unicode.txt

```diff
@@ -1,5 +1,5 @@
-FIXME: See if we can fix this in lit by using Unicode strings.
-REQUIRES: shell
+Windows echo does not have good Unicode support.
+UNSUPPORTED: system-windows
```

Comment:
Does something like `RUN: python3 -c 'print("ようこそ")' | FileCheck %s` work?

Reply:
FTR, the test works fine for me on Windows, unless I force it to use the gnuwin32 `echo` instead of cmd's `echo`.

Maybe use `printf` instead of `echo`? That worked in my environment.